antenna-selection/README.rst

```diff
@@ -16,23 +16,31 @@ Solution: One possible solution is indicated by the red nodes below.
 
 .. image:: readme_imgs/example_solution.png
 
+This problem is an example of an optimization problem, known as the maximum independent set problem.  Our objective is to maximize the number of nodes in a set, with the constraint that no edges be contained in the set.  For the D-Wave System, we can modify this problem to formulate it as a quadratic unconstrained binary optimization problem (QUBO).  There are a wide variety of applications for this problem, such as scheduling and error correcting codes (as shown in [1]_).
+
 Usage
 -----
 To run the demo:
 ::
   python antennas.py
 
+After running, the largest independent set found in the graph will be printed to the command line and two images (.png files) will be created.  An image of the original graph can be found in the file ``antenna_plot_original.png``, and an image of the graph with the nodes in the independent set highlighted in a different color can be found in the file ``antenna_plot_solution.png``. 
+
+To modify the program to consider a different set of antennas locations, modify the graph G.
```

Comment:
This sounds as though you need to modify the algo for a new problem. Maybe just "To run the program on a different problem, modify graph G to represent a different antenna network" ?